packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -23,6 +23,7 @@ import { MarkdownString } from './markdown-string';
 import { MarkdownString as MarkdownStringInterface } from '@theia/core/lib/common/markdown-rendering';
 import { TaskDto } from '../common/plugin-api-rpc';
 import { TaskGroup } from './types-impl';
+import { expect } from 'chai';
```

Comment:
I don't think this import is necessary, we can write our tests with `assert` instead like the rest of the file.